releaseNotes/JavaScriptProvider.md

```diff
@@ -0,0 +1,27 @@
+# Azure App Configuration JavaScript Provider
+
+[Source code][source_code] | [Package (npm)][package] | [Samples][samples]
+
+## 1.0.0-preview.2 - December 15, 2023
+### Breaking Changes
+- The label filter in a selector is restricted to a single label. An error will be thrown if a label filter contains `*` or `,`. This change is to avoid the ambiguity when multiple values are loaded for the same key. Key-values with different labels can still be loaded using separate selectors for proper configuration composition. [#22](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/22)
+
+### Bug Fixes
+- When multiple selectors are provided, the last one takes precedence. If there are selectors with the same key filter and label filter, they are deduplicated. This change ensures that the original selector precedence is maintained as provided. [#23](https://github.com/Azure/AppConfiguration-JavaScriptProvider/issues/23)
+- Updated the reference of `@azure/core-rest-pipeline` to `1.12.2`, addressing [Azure/azure-sdk-for-js#27037](https://github.com/Azure/azure-sdk-for-js/issues/27037). When it fails resolving the given endpoint, the underlying JavaScript SDK now retries before throwing an error.

```

Comment:
It's too much implementation detail to mention "the underlying JavaScript SDK". Does this work?

Updated the reference of `@azure/core-rest-pipeline` to `1.12.2`, which added retry for DNS resolution of App Configuration endpoints. [Azure/azure-sdk-for-js#27037](https://github.com/Azure/azure-sdk-for-js/issues/27037)

Reply:
It looks concise and I'll apply it.